### ISSUE TITLE
AVT 1 - React or the Header Base w/Navigation, Actions & SideNav UI Shell has DAP violations 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3576]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
DAP - July 2019 Ruleset

## Detailed Description
Run DAP on the Header Base w/Navigation, Actions & SideNav UI Shell. The following 7 violations:
![Screen Shot 2019-07-29 at 4 35 01 PM](https://user-images.githubusercontent.com/21676914/62084396-e0b17500-b21e-11e9-85fb-7a714de0845a.png)
 violations are identified (see screenshots below):
![HeaderNavActionsSideNav-1](https://user-images.githubusercontent.com/21676914/62080083-8f50b800-b215-11e9-9c4a-acad6af5fd27.png)
![HeaderNavActionsSideNav-2](https://user-images.githubusercontent.com/21676914/62080124-a4c5e200-b215-11e9-865f-b8619a42b07a.png)

Note:
Add a role to the parent of the menuitems. Use role="menu" or "menubar" or "group" to fix the issue. DAP only reports this issue if the page size reduced and the links do not display across the top of the page.

**Comment by abbeyhrt** [comment]
@snidersd looking into this, it seems the color contrast violation is coming from the SkipToContent component when it's not visible, DAP mistakenly sensing it there with our regular `a` tag color. If you run DAP with the SkipToContent visible, no violation is present: 
![Screen Shot 2019-12-13 at 10 44 49 AM](https://user-images.githubusercontent.com/17281178/70816709-b0b1db00-1d95-11ea-8903-19a2c29b6272.png)
and the "Each page should have no more than one main section..." violation is just coming from our storybook environment, not from the UIShell itself.  If we address the "An element with WAI-ARIA role must be contained within a valid element" violations, could we close this issue?

**Comment by abbeyhrt** [comment]
@snidersd looking into this, it seems the color contrast violation is coming from the SkipToContent component when it's not visible, DAP mistakenly sensing it there with our regular `a` tag color. If you run DAP with the SkipToContent visible, no violation is present: 
![Screen Shot 2019-12-13 at 10 44 49 AM](https://user-images.githubusercontent.com/17281178/70816709-b0b1db00-1d95-11ea-8903-19a2c29b6272.png)
and the "Each page should have no more than one main section..." violation is just coming from our storybook environment, not from the UIShell itself.  If we address the "An element with WAI-ARIA role must be contained within a valid element" violations, could we close this issue?

**Comment by snidersd** [comment]
@abbeyhrt @mattrosno I reported the color contrast violation on the Skip to Main to the [Accessibility Web Engine Github repo # 234](https://github.ibm.com/IBMa/accessibility-web-engine/issues/234). If the violation,  "An element with WAI-ARIA role must be contained within a valid element" is addressed the issue should be closed. Thx.

**Comment by asudoh** [comment]
> If the violation, "An element with WAI-ARIA role must be contained within a valid element" is addressed the issue should be closed.

Thank you for clarifying @snidersd - I don't see such violation in latest code. Closing.
